test/File.js

```diff
@@ -236,13 +236,13 @@ describe('File', function() {
 
       var file = new File(options);
 
-      var copy1 = file.clone({ contents: false });
+      var copy1 = file.clone({contents: false});
```

Comment:
can we adjust this rule?  I think this pattern looks terrible compared to the one we currently use.